src/genie/libs/parser/iosxe/tests/ShowAvcSdServiceInfoSummary/cli/equal/golden_output_expected.py

```diff
@@ -0,0 +1,13 @@
+expected_output = {
+    "active_controller": {
+        "ipv4_address": "10.11.236.21",
+        "status": "Connected",
+        "type": "Primary",
+    },
+    "device": {
+        "id": "sd-sw2.lab.com",
+        "ipv4_address": "10.18.29.33",
```

Comment:
better to use `address` instead of `ipv4_address` based on output. we may see ipv6 address here.

Reply:
If that's the case, I should edit the regex to include IPv4 and IPv6
![image](https://user-images.githubusercontent.com/13859438/96164130-cbd70500-0ee8-11eb-81b1-d07b2652c540.png)

Reply:
Updated